videoanalyst/data/dataset/dataset_impl/trackingnet.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import copy
+from loguru import logger
+import os.path as osp
+from typing import Dict, List
+
+import cv2
+import numpy as np
+from yacs.config import CfgNode
+
+from videoanalyst.data.dataset.dataset_base import TRACK_DATASETS, DatasetBase
+from videoanalyst.evaluation.got_benchmark.datasets import TrackingNet
+from videoanalyst.pipeline.utils.bbox import xywh2xyxy
+
+_current_dir = osp.dirname(osp.realpath(__file__))
+
+
+@TRACK_DATASETS.register
+class TrackingNetDataset(DatasetBase):
+    r"""
+    ILSVRC2015-VID dataset helper
+
+    Hyper-parameters
+    ----------------
+    dataset_root: str
+        path to root of the dataset
+    subset: str
+        dataset split name (train|val|train_val)
+    ratio: float
+        dataset ratio. used by sampler (data.sampler).
+    max_diff: int
+        maximum difference in index of a pair of sampled frames 
+    check_integrity: bool
+        if check integrity of dataset or not
+    """
+    default_hyper_params = dict(
+        dataset_root="datasets/ILSVRC2015",
```

Comment:
replace with right data path?

Reply:
It is the right path. ILSVRC2015-DET and ILSVRC2015-VID share the same root & file structure. I consider to give a description of dataset structure in future.